.github/CODEOWNERS

```diff
@@ -15,10 +15,10 @@
 /tests/modules/javascript                   @aristath @gziolo
 /tests/testdata/modules/javascript          @aristath @gziolo
 
-# Focus: Object Caching
-/modules/object-caching                     @tillkruss @dustinrue
-/tests/modules/object-caching               @tillkruss @dustinrue
-/tests/testdata/modules/object-caching      @tillkruss @dustinrue
+# Focus: Object Cache
+/modules/object-cache                       @tillkruss @spacedmonkey
+/tests/modules/object-cache                 @tillkruss @spacedmonkey
+/tests/testdata/modules/object-cache        @tillkruss @spacedmonkey
```

Comment:
Let's wait with this change just a little more until we've confirmed. If/once we merge this, we also need to update e.g. the spreadsheet with our PoCs accordingly.

Reply:
It's confirmed.

<img width="822" alt="Screen Shot 2022-01-25 at 8 49 57 AM" src="https://user-images.githubusercontent.com/665029/151021669-8bb7c917-d050-4758-adce-fab57a6848d4.png">